docs/make.jl

```diff
@@ -5,17 +5,23 @@ org, reps = :cncastillo, :KomaMRI
 base = "$org/$reps.jl"
 repo_root_url = "https://github.com/$base/blob/master"
 
-# User can add some directories from the literate folder
-foldernames = ["examples"]
-
 # Define some paths
 com = joinpath(dirname(@__DIR__), "assets")
 exa = joinpath(dirname(@__DIR__), "examples")
 src = joinpath(@__DIR__, "src")
 lit = joinpath(exa, "literate")
 gen = joinpath(src, "generated")
 assets = joinpath(src, "assets")
-folderpaths = [joinpath(lit, foldername) for foldername in foldernames]
+exaname = "examples"
+
+# Create empty folders for in the assets directory for the literate generated sections
+mkpath(joinpath(assets, exaname))
+mkpath(joinpath(gen, exaname))
+
+# Copy the logos into the assets folder if necessary
+logo, logo_dark = joinpath(assets, "logo.svg"), joinpath(assets, "logo-dark.svg")
+(!isfile(logo)) && cp(joinpath(com, "logo.svg"), logo)
+(!isfile(logo_dark)) && cp(joinpath(com, "logo-dark.svg"), logo_dark)
```

Comment:
I do not see any other repo doing this.

Reply:
We aim to utilize the same icon file for both the documentation and the GUI. As Documenter searches for a logo file in its assets folder (refer to [this link](https://documenter.juliadocs.org/stable/man/guide/#Adding-a-logo-or-icon)), it is necessary to copy it there initially.

Reply:
Ok